core/controllers/learner_dashboard_test.py

```diff
@@ -278,7 +277,6 @@ def test_get_message_summaries(self):
         response_dict = self.get_json(thread_url)
         messages_summary = response_dict['message_summary_list']
         first_message = messages_summary[0]
-
```

Comment:
Keep this newline, helps maintain readability. The setup stages, followed by data acquisition, followed by assertions. So the newlines help split these logical stages in the test.

Reply:
Done